### PR TITLE
FFI 0.8.0

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

File: releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Info.plist [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	<key>CFBundleExecutable</key>
+	<string>libzcashlc</string>
+	<key>CFBundleIdentifier</key>
+	<string>libzcashlc</string>
+	<key>CFBundleName</key>
+	<string>libzcashlc</string>
+	<key>CFBundlePackageType</key>
+	<string>BNDL</string>
+	<key>CFBundleVersion</key>
+	<string>0.7.5</string>
```
Comment (str4d) [inline releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Info.plist:14]:
```suggestion
	<string>0.8.0</string>
```

File: releases/XCFramework/libzcashlc.xcframework/ios-arm64_x86_64-simulator/libzcashlc.framework/Info.plist [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	<key>CFBundleExecutable</key>
+	<string>libzcashlc</string>
+	<key>CFBundleIdentifier</key>
+	<string>libzcashlc</string>
+	<key>CFBundleName</key>
+	<string>libzcashlc</string>
+	<key>CFBundlePackageType</key>
+	<string>BNDL</string>
+	<key>CFBundleVersion</key>
+	<string>0.7.5</string>
```
Comment (str4d) [inline releases/XCFramework/libzcashlc.xcframework/ios-arm64_x86_64-simulator/libzcashlc.framework/Info.plist:14]:
```suggestion
	<string>0.8.0</string>
```

File: releases/XCFramework/libzcashlc.xcframework/macos-arm64_x86_64/libzcashlc.framework/Info.plist [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	<key>CFBundleExecutable</key>
+	<string>libzcashlc</string>
+	<key>CFBundleIdentifier</key>
+	<string>libzcashlc</string>
+	<key>CFBundleName</key>
+	<string>libzcashlc</string>
+	<key>CFBundlePackageType</key>
+	<string>BNDL</string>
+	<key>CFBundleVersion</key>
+	<string>0.7.5</string>
```
Comment (str4d) [inline releases/XCFramework/libzcashlc.xcframework/macos-arm64_x86_64/libzcashlc.framework/Info.plist:14]:
```suggestion
	<string>0.8.0</string>
```

**Review by str4d — COMMENTED** [review]
The source plist version was updated, but not the actual ones that get used (because the source plist update was done after `make xcframework`).

**Review by str4d — COMMENTED** [review]
utACK 9314c83d7a09d88e1c0bd3ff3738a50833325059

**Review by daira — COMMENTED** [review]
Post-hoc ACK.

**Pull request merged**